src/app/services/security/http-interceptor.service.ts

```diff
@@ -21,9 +21,24 @@ export class HttpinterceptorService implements HttpInterceptor {
     const tempToken = auth.getToken();
 
     if (tempToken != null) {
-      const afterTokenreq: HttpRequest<any> = req.clone({
-        setHeaders: { authorization: tempToken },
+      const afterTokenreq: HttpRequest<any>;
```

Comment:
Use `let` instead of `const` so that you can assign a value later.

Reply:
OK